# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/moz.build
# Commit: 745df1a31541
# Full Hash: 745df1a31541f1f0b6e31f1b8d7414bdbb2f1e83
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-03-06 04:41:40
# Regressor Bug: 1858958
# File Overlap Count: 1
# Description:
#   Bug 1858958 - WebIDL file and skeleton code for AudioData. r=chunmin,webidl,saschanaz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D190943
# ==============================================================================

diff -r adf17fc47c84 -r 745df1a31541 dom/media/webcodecs/moz.build
--- a/dom/media/webcodecs/moz.build	Tue Mar 05 15:49:24 2024 +0000
+++ b/dom/media/webcodecs/moz.build	Tue Mar 05 15:52:11 2024 +0000
@@ -21,6 +21,7 @@
 ]
 
 EXPORTS.mozilla.dom += [
+    "AudioData.h",
     "DecoderTemplate.h",
     "DecoderTypes.h",
     "EncodedVideoChunk.h",
@@ -35,6 +36,7 @@
 ]
 
 UNIFIED_SOURCES += [
+    "AudioData.cpp",
     "DecoderAgent.cpp",
     "DecoderTemplate.cpp",
     "EncodedVideoChunk.cpp",